docs/source/template_options.rst

```diff
@@ -0,0 +1,266 @@
+1. Simple or custom installation
```

Comment:
Add another heading above this, as the page is named "1. Simple or custom installation"

Reply:
This bugged me too while I was coding, but adding a heading above would display it in the sidebar. 

IMO, clarity of navigation via UI elements like the sidebar beats out the page title (though this is biased by recently seeing a let's-do-this-fast scientist scanning the sidebar to try to speed run installation, rather than read through even the overview).

That said, open to further discussion of alternatives. (Part of me is tempted to do some css kung fu here to get the sidebar and title how we want it, but, well, probably not the best way.)